transport-native-epoll/src/main/c/netty_epoll_linuxsocket.c

```diff
@@ -156,10 +157,12 @@ static void netty_epoll_linuxsocket_joinGroup(JNIEnv* env, jclass clazz, jint fd
     struct ip_mreq mreq;
 
     if (netty_unix_socket_initSockaddr(env, groupAddress, scopeId, 0, &groupAddr, &groupAddrSize) == -1) {
+        netty_unix_errors_throwIOException(env, "Could not init sockaddr");
```

Comment:
can you add context to these exception so they can be distinguished? For example:

> Could not init sockaddr for groupAddress

> Could not init sockaddr for interfaceAddress

Reply:
(general comment)